src/main/java/com/gluonhq/substrate/util/ProcessRunner.java

```diff
@@ -64,6 +64,7 @@
     private boolean interactive;
 
     private static Path processLogPath;
+    private static boolean ciEnvironment;
```

Comment:
I think this is not really specific to CI. I do understand the idea, but it might be better to rename it to "consoleProcessLogging" or so.

Reply:
Makes sense, renamed